std_msgs/Header header
float32 temp_fet
float32 temp_motor
float32 current_in
float32 pid_pos_now